modules/cluster-logging-deploy-cli.adoc

```diff
@@ -390,7 +390,7 @@ You should see several pods for OpenShift Logging, Elasticsearch, Fluentd, and K
 +
 [source,terminal]
 ----
-$ oc get pods -n openshift-logging
+$ oc get pods -l component=collector -n openshift-logging
```

Comment:
This will only display the collector pods.  Is that the intent of the change.  Text would indicate all logging pods